src/cloud-cli/meltano/cloud/api/config.py

```diff
@@ -102,6 +107,84 @@ def config_path(self) -> Path:
 
         return self._config_path
 
+    @staticmethod
+    def decode_jwt(token: str) -> dict[str, t.Any]:
+        """Decode a JWT without verifying.
+
+        Args:
+            token: the jwt to decode
+
+        Returns:
+            The decoded jwt.
+        """
+        return jwt.decode(token, options={"verify_signature": False})
+
+    @property
+    def trks_and_pids(self) -> t.List[str]:
+        """Get tenant resource keys and project ids from id token.
+
+        Returns:
+            List of trks and pids in the form '<trk>::<jpid>`
+
+        """
+        with suppress(jwt.DecodeError):
+            decoded = self.decode_jwt(self.id_token)
+            trks_and_pids = decoded.get("custom:trk_and_pid")
+            if trks_and_pids:
+                return [perm.strip() for perm in trks_and_pids.split(",")]
+        return []
```

Comment:
I think we should avoid exposing this beyond this class by prefixing its name with an underscore.